docs/content/users/usage/faq.md

```diff
@@ -300,6 +300,8 @@ Delete it and migrate it to a new project with your preferred name:
 4. Start the new project with `ddev start`.
 5. Import the database dump from step one: `ddev import-db --file=/path/to/db.sql.gz`.
 
+In rare cases, projects might fail to start after renaming if legacy add-ons depended on your previous project's name. From the project directory, run `mv .ddev/traefik .ddev/traefik.bak` to clear and backup the Traefik configuration (does not lose any data).
```

Comment:
Actually, I think `rm -r .ddev/traefik/` is almost always in order when renaming, unless there's custom config (rare) in which case it will have to be adapted.